### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by pivovarit**

### CODE REVIEW

**Comment by mesutcelik** [comment]
Are we going to use this sample as [getting started guide](https://github.com/hazelcast-guides/hazelcast-quarkus)? Then we should mention it in the document.

**Comment by pivovarit** [comment]
Eventually yes - but we need to release the extension first to be able to use it in the guide

**Comment by mesutcelik** [comment]
Good. I suggest to create a github issue mentioning `update doc with guide` for the follow-up.

Can you also mention Hazelcast 4+ Support in the readme?

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# quarkus-hazelcast-client
+# Hazelcast Client for Quarkus
+
+<a href="https://github.com/actions/toolkit"><img alt="GitHub Actions status" src="https://github.com/hazelcast/quarkus-hazelcast-client/workflows/build/badge.svg"></a>
```
Comment (leszko) [inline README.md]: It does not display correctly for me.
image

**Comment by pivovarit** [comment]
I think we'll revise it a few times still

**Pull request merged**